test/unit/math/mix/scal/fun/falling_factorial_test.cpp

```diff
@@ -1,158 +1,22 @@
-#include <stan/math/mix/scal.hpp>
-#include <gtest/gtest.h>
-#include <boost/math/special_functions/digamma.hpp>
-#include <test/unit/math/rev/scal/fun/util.hpp>
-#include <test/unit/math/mix/scal/fun/nan_util.hpp>
+#include <test/unit/math/test_ad.hpp>
+#include <limits>
 
-TEST(AgradFwdFallingFactorial, FvarVar_1stDeriv) {
-  using stan::math::digamma;
-  using stan::math::falling_factorial;
-  using stan::math::fvar;
-  using stan::math::var;
+TEST(mathMixScalFun, fallingFactorial) {
+  auto f = [](const int x2) {
+    return
+        [=](const auto& x1) { return stan::math::falling_factorial(x1, x2); };
+  };
+  stan::test::expect_ad(f(-2), -3.0);  // throws
 
-  fvar<var> a(5.0, 1.0);
-  fvar<var> c = falling_factorial(a, 3);
+  stan::test::expect_ad(f(3), 5);
 
-  EXPECT_FLOAT_EQ(falling_factorial(5, 3), c.val_.val());
-  EXPECT_FLOAT_EQ(
-      falling_factorial(5, 3) * (digamma(5 + 1) - digamma(5 + 1 - 3)),
-      c.d_.val());
+  // 3rd order derivatives near zero
+  stan::test::ad_tolerances tols;
```

Comment:
Do you think we should keep a list somewhere of functions that require changes to the finite difference tolerances to test?

Like in an issue?

Reply:
I don't see any reason to.  Most functions would require changes to tolerances if values got more extreme.  We could try to evaluate analytical changes here, but really it's a problem with the way the framework does relative value comparison for values close to zero.  It's a hard problem in general.

Reply:
Cool. They'll be easy to find if we ever need to via grepping.